tests/test_setup.py

```diff
@@ -42,6 +57,8 @@ def get_args(yaml_path: str = '', cwl_inline_subworkflows: bool = False) -> argp
                     for yml_namespace, yml_paths_dict in yml_paths.items()
                     for yml_path_str, yml_path in yml_paths_dict.items()]
 
+yml_paths_partial_failure = [('examples/test_rand_fail.yml', Path('examples/test_rand_fail.yml'))]
```

Comment:
We should try not to hard-code paths to specific workflow files. We should use the [auto-discovery](https://workflow-inference-compiler.readthedocs.io/en/latest/userguide.html#auto-discovery) mechanism, together with a config_ci.json file to blacklist this file from the regular CI and create a new category for workflows which are expected to fail. For now, this is okay, but we should make the change in a future PR.

Reply:
Yes I will move this to auto-discovery in a future PR